### ISSUE TITLE
Test easyCampaign

### ISSUE THREAD

**Issue by mikemhenry** [OpenFreeEnergy/ExampleNotebooks#51]
We don't test this example but we should. I'm thinking some `sed` modifications to reduce the time it takes and then we run the script and make sure it doesn't throw an error

**Comment by dwhswenson** [comment]
TBH, I think we might want to remove the scripts and just point to the tutorial. If you can come up with a  way to strip out the bash parts of the tutorial and run them, that would be really useful. (Some Markdown parser might allow it? Or you could probably even get it from a DOM approach, if I was correctly marking the relevant blocks as bash.)

You'll also have to mock `sbatch` as just running in bash or something.

**Comment by mikemhenry** [comment]
lol `alias sbatch=bash` what could go wrong? (this is how I would mock in CI)

We should probably add a few more `sbatch` preamble things like `#SBATCH --job-name=openfe_easy_campaign` and `#SBATCH --ntasks=1` or something since while it is probably the smallest example you can make, I don't know many systems where that would work on, most make you put the group you are charging or your email...

Or we can also wait until someone says "hey I ran this and I need some help adding some bits" and we can just do it then.

And I can see us forgetting to update a script but we did update the tutorial, so unless we test it all, we are going to get hit with some API drift.

**Comment by dwhswenson** [comment]
> We should probably add a few more sbatch preamble things like #SBATCH --job-name=openfe_easy_campaign and #SBATCH --ntasks=1 or something since while it is probably the smallest example you can make, I don't know many systems where that would work on, most make you put the group you are charging or your email...

I don't think it is worth making that SLURM setup stuff much more detailed. If we're going to go too far that direction, let's do it right and talk about template files. But I think that's outside the scope of what we should actually be doing here. I'd say wait until someone needs help.

> And I can see us forgetting to update a script but we did update the tutorial, so unless we test it all, we are going to get hit with some API drift.

This is part of why I say we probably want to remove the scripts. The same information is in the tutorial. The scripts were a good starting point and necessary before the tutorial, but tutorial should be single source of truth.
